### ISSUE TITLE
[bug/regression] seeing TypeError: Cannot read property 'func' of undefined with v1.8.1

### ISSUE THREAD

**Issue by koulmomo** [wix/eslint-plugin-lodash#74]
console output:

```sh
Cannot read property 'func' of undefined
TypeError: Cannot read property 'func' of undefined
    at isIterateeParamDefinition (/Users/koulmomo/americorp/foo-app/node_modules/eslint-plugin-lodash/lib/rules/prefer-times.js:47:166)
    at EventEmitter.Identifier (/Users/koulmomo/americorp/foo-app/node_modules/eslint-plugin-lodash/lib/rules/prefer-times.js:54:18)
    at EventEmitter.emit (events.js:117:20)
    at NodeEventGenerator.enterNode (/Users/koulmomo/americorp/foo-app/node_modules/eslint/lib/util/node-event-generator.js:42:22)
    at CodePathAnalyzer.enterNode (/Users/koulmomo/americorp/foo-app/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:609:23)
    at CommentEventGenerator.enterNode (/Users/koulmomo/americorp/foo-app/node_modules/eslint/lib/util/comment-event-generator.js:99:23)
    at Controller.traverser.traverse.enter (/Users/koulmomo/americorp/foo-app/node_modules/eslint/lib/eslint.js:887:36)
    at Controller.__execute (/Users/koulmomo/americorp/foo-app/node_modules/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/koulmomo/americorp/foo-app/node_modules/eslint/node_modules/estraverse/estraverse.js:501:28)
    at Controller.Traverser.controller.traverse (/Users/koulmomo/americorp/foo-app/node_modules/eslint/lib/util/traverser.js:38:33)
```

.eslintrc
```json
{
  "root": true,
  "parser": "babel-eslint",

  "extends": [
    "plugin:lodash/recommended"
  ],

  "plugins": [
    "lodash"
  ],

  "rules": {
    "func-style": 1,
    "indent": [2, 2, {"SwitchCase": 1}],
    "max-len": [2, 180, 2, {"ignoreUrls": true, "ignoreComments": true}],
    "max-statements": 1,
    "no-var": 1,
    "object-shorthand": 1,
    "prefer-spread": 1,
    "prefer-template": 1,
    "strict": 1,

    "lodash/prefer-lodash-method": 0,
    "lodash/prefer-constant": 1,
    "lodash/path-style": 1,
    "lodash/prefer-lodash-chain": 0
  }
}
```

`eslint@2.8.0`
`babel-eslint@6.0.4`

**Comment by mik01aj** [comment]
Temporary workaround: disable the `prefer-times` rule.

**Comment by ganimomer** [comment]
Fixed in `v1.8.2`. (Forgot to tag the issue in the [commit](https://github.com/wix/eslint-plugin-lodash/commit/c199ed5f3c055648791079142c9afdee666ffbed) message this time, sorry!)
